cmd/monitor/monitor.go

```diff
@@ -121,10 +134,13 @@ func main() {
 		if err = cleanupWorkspace(config, clients); err != nil {
 			fmt.Printf("Failed to remove project: %v\n", err)
 		}
-		if elapsed := time.Since(startTime); elapsed < interval {
-			fmt.Printf("Sleeping for %v before next iteration\n", interval-elapsed)
-			time.Sleep(interval - elapsed)
+		// sleep a minimum of 30 seconds between runs
+		toSleep := 30 * time.Second
+		if elapsed := time.Since(startTime); interval-elapsed > toSleep {
+			toSleep = interval - elapsed
 		}
+		fmt.Printf("Sleeping for %v before next iteration\n", toSleep)
```

Comment:
Please use glog to control log spam with verbosity levels

Reply:
Yeah, this is on me. Joel said he'd start adding FIXME comments here, which I'll replace once I get to the glog stuff.